.gitignore

```diff
@@ -92,5 +92,5 @@ src/scss/_themes.scss
 # Eleventy
 # We generate our own .eleventyignore file dynamically during builds
 .eleventyignore
-
+instance/mydatabase.db
```

Comment:
We might want to allow db commits. Especially if we modify the structure at all.